axiom/cmd_appinfo_transmit.c

```diff
@@ -397,7 +398,9 @@ void nr_cmd_appinfo_process_event_harvest_config(const nrobj_t* config,
       harvest_limits, "error_event_data", NR_MAX_ERRORS);
   app_limits->span_events = nr_cmd_appinfo_process_get_harvest_limit(
       harvest_limits, "span_event_data",
-      NR_DEFAULT_SPAN_EVENTS_MAX_SAMPLES_STORED);
+      0 == info.span_events_max_samples_stored
+          ? NR_MAX_SPAN_EVENTS_MAX_SAMPLES_STORED
```

Comment:
Currently NR_MAX_SPAN_EVENTS_MAX_SAMPLES_STORED is 10000.  My understanding based on testing is this value is too large.  Where will the more realistic max value determined in testing be enforced?

Reply:
It's too large for DT only.  For infinite tracing it is not an issue.